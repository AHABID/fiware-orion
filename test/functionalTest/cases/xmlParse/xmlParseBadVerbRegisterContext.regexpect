HTTP/1.1 405 Method Not Allowed
Content-Length: 0
Allow: POST
Date: REGEX(.*)

++++++++++++++++++++
