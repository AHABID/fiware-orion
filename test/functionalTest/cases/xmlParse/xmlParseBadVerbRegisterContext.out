HTTP/1.1 405 Method Not Allowed
Content-Length: 0
Allow: POST
Date: Sun, 01 Jun 2014 17:45:28 GMT

++++++++++++++++++++
